mono/mini/debugger-agent.c

```diff
@@ -12080,11 +12104,11 @@ unity_process_breakpoint_inner(DebuggerTlsData *tls, gboolean from_signal, Il2Cp
 	}
 
 	if (ss_reqs->len > 0)
-		ss_events = create_event_list(EVENT_KIND_STEP, ss_reqs, ji, NULL, &suspend_policy);
```

Comment:
should this be #if'd out?

Reply:
It's in unity_process_breakpoint_inner(), it already is